runtime/near-vm-logic/src/dependencies.rs

```diff
@@ -36,7 +36,8 @@ pub enum ExternalError {
     InvalidAccountId,
     InvalidMethodName,
     InvalidPublicKey,
-    StorageError,
+    /// Error from underlying storage, serialized
+    StorageError(Vec<u8>),
```

Comment:
Is the reason we serialize it here is to cut the dependency from the primitives?